communication/best-practices.md

```diff
@@ -0,0 +1,85 @@
+# Mailing list and Github usage best practice                                                       
+                                                                                
+The Kubernetes Mailing list or Google Groups functions as the primary means of  
+asynchronous communication for the project's                                    
+[Special Interest Groups (SIG)][sig-list] and [Working Groups (WG)][sig-list].  
+That's why you may want to set your filters in your Gmail account to attain a
```

Comment:
Dawn has a good point! Let's add a note here with something like "If you use Gmail..."